test/js/specs/unit/views/maps/ViewfinderView.spec.js

```diff
@@ -0,0 +1,199 @@
+define([
+  "views/maps/ViewfinderView",
+  "models/maps/Map",
+  // The file extension is required for files loaded from the /test directory.
+  "/test/js/specs/unit/views/maps/ViewfinderViewHarness.js",
+  "/test/js/specs/shared/clean-state.js",
+  "/test/js/specs/shared/create-spy.js",
+], (ViewfinderView, Map, ViewfinderViewHarness, cleanState, createSpy) => {
+  const should = chai.should();
+  const expect = chai.expect;
+
+  describe("ViewfinderView Test Suite", () => {
+    const state = cleanState(() => {
+      const view = new ViewfinderView({ model: new Map() });
+      const spy = createSpy();
+      view.model.zoomTo = spy;
+      const harness = new ViewfinderViewHarness(view);
+
+      return { harness, spy, view };
+    }, beforeEach);
+
+    it("creates a ViewfinderView instance", () => {
+      state.view.should.be.instanceof(ViewfinderView);
+    });
+
+    it("has an input for the user's search query", () => {
+      state.view.render();
+
+      state.harness.typeQuery("123")
```

Comment:
This is so cool and will be super useful for writing tests for other views!!

Reply:
I noticed there are no *View.spec.js files, so hopefully this is a useful pattern to get us started. Using a "harness" is a great way to hide the implementation details, especially when writing tests for reusable components. this way if the class name of something changes, we'll only need to update the harness, rather than every single test that uses the reusable view.